bedrock/firefox/templates/firefox/new/batm/private.html

```diff
@@ -2,18 +2,20 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
+{% add_lang_files "firefox/new/batm" %}
+
 {% from "macros.html" import google_play_button with context %}
```

Comment:
Yeah, I can remove it from all the BATM variants, just didn't want to touch every file if I didn't have to. But if it's just this deletion that shouldn't be too bad.